stdlib/std.ncl

```diff
@@ -909,8 +909,8 @@
 
     label
       | doc m%"
-          The label submodule gathers functions that manipulate the label of
-          a contract.
+          The label submodule provides functions that manipulate the label of a
+          contract.
```

Comment:
I'm not sure what else to write here, but I feel like a label doesn't really belong to a contract, but to a contract application.

Reply:
Ah, you're right indeed. I guess saying exactly that is fine.